docs/experiments-ciral-v1.0.md

```diff
@@ -0,0 +1,139 @@
+- [Somali](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-so.20230721.e850ea.tar.gz)
+- [Swahili](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-sw.20230721.e850ea.tar.gz)
+- [Yoruba](https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.ciral-v1.0-yo.20230721.e850ea.tar.gz)
+
+
+## Data Prep
+
+We would need the topic and qrel files from the [Hugging Face repo](https://huggingface.co/datasets/CIRAL/ciral) to reproduce the baselines.  
+
+Clone the repo and copy the files to `tools/topics-and-qrels` in the cloned `Pyserini` repo. This requires the Pyserini [dev install](https://github.com/castorini/pyserini/blob/master/docs/installation.md#development-installation).
+
+```bash
+git clone https://huggingface.co/datasets/CIRAL/ciral
+cp -r ciral/*/*/* $PYSERINI_PATH/tools/topics-and-qrels/
+```
+
+
+
+## Retrieval: Dev Topics
+
+```bash
+python -m pyserini.search.lucene --index ciral-v1.0-ha \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-ha-train \
+    --output runs/run.ciral-v1.0-ha.bm25.topics.ciral-v1.0-ha.train.txt \
+    --pretokenized --language ha \
+    --batch 128 --threads 16 --bm25 --hits 1000
+
+python -m pyserini.search.lucene --index ciral-v1.0-so \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-so-train \
+    --output runs/run.ciral-v1.0-so.bm25.topics.ciral-v1.0-so.train.txt \
+    --pretokenized --language so \
+    --batch 128 --threads 16 --bm25 --hits 1000
+
+python -m pyserini.search.lucene --index ciral-v1.0-sw \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-sw-train \
+    --output runs/run.ciral-v1.0-sw.bm25.topics.ciral-v1.0-sw.train.txt \
+    --pretokenized --language sw \
+    --batch 128 --threads 16 --bm25 --hits 1000
+
+python -m pyserini.search.lucene --index ciral-v1.0-yo \
+    --topics tools/topics-and-qrels/topics.ciral-v1.0-yo-train \
+    --output runs/run.ciral-v1.0-yo.bm25.topics.ciral-v1.0-yo.train.txt \
+    --pretokenized --language yo \
+    --batch 128 --threads 16 --bm25 --hits 1000
+
+```
+
+## Evaluation: Dev Topics
+```bash
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-ha-train.tsv runs/run.ciral-v1.0-ha.bm25.topics.ciral-v1.0-ha.train.txt
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-so-train.tsv runs/run.ciral-v1.0-so.bm25.topics.ciral-v1.0-so.train.txt
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-sw-train.tsv runs/run.ciral-v1.0-sw.bm25.topics.ciral-v1.0-sw.train.txt
+python -m pyserini.eval.trec_eval -c -m recall.1000 tools/topics-and-qrels/qrels.ciral-v1.0-yo-train.tsv runs/run.ciral-v1.0-yo.bm25.topics.ciral-v1.0-yo.train.txt
+```
+
+## Effectiveness
+
+### Yoruba
+
+With the above commands, you should be able to reproduce the following results:
+| **recall@1000**                                                                   | **BM2 (default)**| 
```

Comment:
Should be BM25.
Why not evaluate also with MRR or nDCG?

Reply:
Hi Jimmy,
Do you think we should hold on about this based on the results?